Sources/MapboxMaps/Foundation/MapView+Attribution.swift

```diff
@@ -0,0 +1,64 @@
+@_implementationOnly import MapboxCommon_Private
+
+@available(iOSApplicationExtension, unavailable)
+extension MapView: AttributionDialogManagerDelegate {
+    func viewControllerForPresenting(_ attributionDialogManager: AttributionDialogManager) -> UIViewController {
+        // Consider delegating out to a delegate
+        return parentViewController!
+    }
+
+    func attributionDialogManager(_ attributionDialogManager: AttributionDialogManager, didTriggerActionFor attribution: Attribution) {
+        let url: URL
+
+        if attribution.isFeedbackURL {
+            url = mapboxFeedbackURL()
+        } else {
+            url = attribution.url
+        }
```

Comment:
```suggestion
        let url: URL = attribution.isFeedbackURL ? mapboxFeedbackURL() : attribution.url
```

Seems like a good opportunity to use the ternary operator